geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -463,6 +463,15 @@ public void setShutDownAllStatus(int newStatus) {
 
   private final Object indexLock = new Object();
 
+  private Set<ColocationCompleteListener> colocationCompleteListeners = new HashSet<>();
+
+  public void setColocationCompleteListener(ColocationCompleteListener colocationCompleteCallBack) {
```

Comment:
change name to add instead of set?  Since we can "add" more listeners and not necessarily "set" a single listener?